docs/examples/frameworks/closed_single_first_stage.json

```diff
@@ -0,0 +1,66 @@
+{
+  "uri": "https://example.com/ocds/ocds-r6ebe6-9af22e2f-5307-4e98-bff6-1ebffbff343d/2019_S197-478648",
+  "version": "1.1",
+  "publishedDate": "2019-10-09T00:00:00Z",
+  "publisher": {
+    "name": "Open Contracting Partnership"
+  },
+  "license": "https://creativecommons.org/publicdomain/zero/1.0/",
+  "publicationPolicy": "http://example.com/publicationPolicy",
+  "extensions": [
+    "https://raw.githubusercontent.com/open-contracting-extensions/ocds_techniques_extension/master/extension.json"
+  ],
+  "releases": [
+    {
+      "ocid": "ocds-r6ebe6-9af22e2f-5307-4e98-bff6-1ebffbff343d",
+      "id": "2019/S 197-478648",
+      "date": "2019-10-09T00:00:00Z",
+      "tag": [
+        "tender"
+      ],
+      "initiationType": "tender",
+      "buyer": {
+        "name": "The Common Services Agency (more commonly known as NHS National Services Scotland) (‘NSS’)",
+        "id": "GB-NHS-YA7"
+      },
+      "parties": [
+        {
+          "name": "The Common Services Agency (more commonly known as NHS National Services Scotland) (‘NSS’)",
+          "id": "GB-NHS-YA7",
+          "identifier": {
+            "scheme": "GB-NHS",
+            "id": "YA7"
+          },
+          "roles": [
+            "buyer"
+          ]
+        }
+      ],
+      "tender": {
+        "techniques": [
```

Comment:
`techniques` is an object, not an array. I suggest validating all the examples in the DRT.

Reply:
done!